addon/components/validated-input/types/select.js

```diff
@@ -69,10 +69,48 @@ export default class SelectComponent extends Component {
     return groups;
   }
 
+  findOption(target) {
+    const targetPath = this.args.optionTargetPath;
+    const valuePath = this.args.optionValuePath || targetPath;
+    let options = this.args.options;
+
+    //flatten pre grouped options
+    if (this.hasPreGroupedOptions) {
+      options = options.flatMap((group) => group.options);
+    }
+
+    // multi select
+    if (this.args.multiple) {
+      const selectedValues = [];
+      for (const option of target.options) {
+        if (option.selected) {
+          selectedValues.push(option.value);
+        }
+      }
```

Comment:
cosnt selectedValues = target.options.filter(option => option.selected)

Reply:
:sparkler: It's beautiful:
```js
Array.prototype.filter
        .call(target.options, option => option.selected)
        .map(option => option.value);
```